doc/autoscale/experiment/mnist/train.py

```diff
@@ -0,0 +1,163 @@
+from PIL import Image
```

Comment:
Do not copy this file, just reuse the file under `demo`

Reply:
Very much agree that it's good to avoid duplicate code. However the `demo/train.py` will perhaps change for different reason and at different time comparing to `mnist/train.py`.
For example, `mnist/train.py` probably will never change after the experiment, but the `demo/train.py` maybe will keep evolving (and breaking the experiment code).
Given that `demo/train.py` change for different reason and at different times, maybe they are not duplicate code? (even they currently have exact same code, but in the future they will diverge, because they change for different reason and at different times).

Reply:
Agree!

Reply:
Mabye we can change the folder name for a general name, such as `demo/job/...`, or someone will feel uncertain about the different this `experiemnt/mnist` and `demo/mnist`.